assets/css/application.scss

```diff
@@ -10,6 +10,46 @@ body {
   font: 14px/1.4 "Helvetica Neue", Helvetica, Arial, sans-serif;
 }
 
+@media (prefers-color-scheme: dark) {
+  body {
+    background: #212121;
+    color: #d0c8c1;
+  }
+  
+  h1, h2, h3, h5 {
+    color: #dadada !important;
+  }
+  
+  .site-footer {
+    color: #ccc !important; 
+  }
+  
+  .site-footer a {
+    color: #ddd !important; 
+  }
+  
+  .home h2 {
+    color: #c7cdce !important;
+  }
+  
+  .license-body pre {
+	background-color: #131313 !important;
+	border: 1px solid #3e3e3e !important;
```

Comment:
Just a nitpick, but these two lines are indented with tabs while the others are indented with spaces. (The same happens in line 41, 45 and 49.)